src/java.base/windows/classes/sun/nio/fs/WindowsFileCopy.java

```diff
@@ -301,7 +301,7 @@ static void move(WindowsPath source, WindowsPath target, CopyOption... options)
                 continue;
             }
             if (option == null) throw new NullPointerException();
-            throw new UnsupportedOperationException("Unsupported copy option");
+            throw new UnsupportedOperationException("Unsupported move option: " + option);
```

Comment:
The parameter is a CopyOption so changing the message to "move option" might be confusion, maybe just change it to "option"?

Reply:
Plain "option" might be best here.